app/src/main/AndroidManifest.xml

```diff
@@ -6,14 +6,16 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.CAMERA" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
i guess we can leave READ_EXTERNAL_STORAGE but add a max sdk version 
```
<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"
                 android:maxSdkVersion="32" />
```

Reply:
Yes, you are right. Otherwise it may crash on older versions